post/controller.js

```diff
@@ -5,29 +5,29 @@ const express = require('express');
 const router = express.Router();
 const urlencodedParser = bodyParser.urlencoded({ extended: false });
 
-router.get(['/', '/posts'], (req, res) => {
+router.get('/posts', (req, res) => {
   postService.getAllPosts().then((posts) => {
-    res.render('posts', { posts });
```

Comment:
Is there a way to make the `_id` field more official? It's a bit weird having the underscore.

Reply:
Nothing I could find.